gh/os-mappings

```diff
@@ -0,0 +1,3 @@
+windows=windows
+mac=macOS
+*=Linux
```

Comment:
*Linux* should be lowercase (*linux*) as I see in releases.

Reply:
Thank you for pointing that out! Not sure how I wrote linux in uppercase...